app/models/repository/validations.rb

```diff
@@ -118,9 +113,10 @@ def validate(record)
 
   class SourceTypeValidator < ActiveModel::Validator
     def validate(record)
-      if record.mirror? && !record.source_type.present?
+      if record.remote? && !record.source_type.present?
         record.errors[:source_address] = 'not a valid remote repository '\
-          "or not accessible (types supported: #{SOURCE_TYPES.join(', ')})"
+          'or not accessible '\
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.